pom.xml

```diff
@@ -0,0 +1,48 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <artifactId>parent</artifactId>
+    <groupId>org.igniterealtime.openfire</groupId>
+    <version>4.3.0-SNAPSHOT</version>
+  </parent>
+  <groupId>org.igniterealtime.openfire.fastpath</groupId>
+  <artifactId>webchat</artifactId>
+  <version>2.6.2</version>
+  <packaging>pom</packaging>
+  <name>Fastpath Webchat</name>
+  <description>Web based chat client for Fastpath</description>
+
+  <properties>
+    <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+
+        <!-- Versions -->
+        <!-- Note; the following jetty.version should be identical to the jetty.version in plugins/pom.xml -->
+    <jetty.version>9.4.11.v20180605</jetty.version>
+    <smack.version>4.2.3</smack.version>
```

Comment:
I think you could probably bump to 4.2.4 for free :)

Reply:
Did bump it to 4.2.4.